index.js

```diff
@@ -0,0 +1,89 @@
+import BookCollection from './modules/bookCollection.js';
+import { storageAvailable, populateStorage, setInputs } from './modules/localStorage.js';
+import displayCurrentDate from './modules/currentDate.js';
+
+const bookCollection = new BookCollection();
+
+const addBookButton = document.querySelector('#add-book');
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+const bookForm = document.querySelector('form');
+const mainSection = document.querySelector('#main-section');
+const empty = document.querySelector('.empty');
+const navLinks = document.querySelectorAll('.nav-link');
+const content = document.querySelectorAll('.content');
+const currentDate = document.querySelector('#date');
+
+displayCurrentDate(currentDate);
+
+if (storageAvailable('localStorage')) {
+  if (!localStorage.getItem('books')) {
+    populateStorage(bookCollection);
+  } else {
+    setInputs(bookCollection);
+  }
+}
+
+/* eventlistners */
+
+function displayBooks() {
+  if (document.querySelector('.books')) {
+    mainSection.removeChild(document.querySelector('.books'));
```

Comment:
- [ ] Why not make use of arrow syntax for functions as arrow functions are part of ES6 and it is a requirement in this project.